error-reporting/api/diagnostics/README.md

```diff
@@ -1,8 +1,15 @@
 # Stackdriver Error Reporting Sample
 
-A sample demonstrating how to use the Google Cloud Stackdriver Diagnostics ASP.NET NuGet package
+A sample demonstrating how to use the Google Cloud Stackdriver Diagnostics ASP.NET Core NuGet package
 to log all of an ASP.NET web application's errors to the Error Reporting page in the Cloud Platform Console.
 
+This sample requires [.NET Core 2.0](
+    https://www.microsoft.com/net/core) or later.  That means using
+[Visual Studio 2017](
+    https://www.visualstudio.com/), or the command line.  Visual Studio 2015 users

```

Comment:
nit: perhaps, either remove one space or better move it to the new line. Also worth adding that we support Visual Studio Code, like I mentioned in other similar comment earlier.